cloudformation/cw-dashboard-substack.cfn.yaml

```diff
@@ -0,0 +1,241 @@
+Conditions:
+  CreateEFSSubstack: !Equals
```

Comment:
Let's use a "logical order" for the conditions and for the parameters, as described before.

Reply:
Done